framework/src/modules/network/migrations/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const path = require('path');
+
+const migrations = [
+	path.join(path.dirname(__filename), './sql/20160723182900_create_schema.sql'),
```

Comment:
do we need to list it one by one?

Reply:
We could read the folder and build the array from all files inside.
I don't have preferences. Let me know if you do.

Reply:
I think it's a bit easier to manage, as we don't need to add the file then create the entry